RealEstate/urls.py

```diff
@@ -19,5 +19,8 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('RealEstate.apps.core.urls')),
+    url(r'^$', include('RealEstate.apps.core.urls'), name='home'),
```

Comment:
This line isn't quite right.  URLconf's with include use namespaces, while actual views use names.  But lets just circumvent the include by moving those URLconfs into this file.  I think our URL scheme is simple enough where this shouldn't be a problem.